src/current/v23.1/migration-strategy-lift-and-shift.md

```diff
@@ -0,0 +1,142 @@
+
+- [Migrate with AWS Database Migration Service (DMS)](aws-dms.html)
+- [Migrate and Replicate Data with Qlik Replicate](qlik.html)
+- [Migrate and Replicate Data with Striim](striim.html)
+
+{{site.data.alerts.callout_info}}
+Depending on the migration service you choose, [long-running transactions](query-behavior-troubleshooting.html#hanging-or-stuck-queries) can occur.  In some cases, these queries will cause [transaction retry errors](common-errors.html#restart-transaction). If you encounter these errors while migrating to CockroachDB using a managed migration service, please reach out to our [Support Resources](support-resources.html).
+{{site.data.alerts.end}}
+
+<a name="data-formats"></a>
+
+### Decide which data formats and storage media you will use
+
+It's important to decide which data formats, storage media, and database features you will use to migrate your data.
+
+Data formats that can be imported by CockroachDB include:
+
+- [SQL](schema-design-overview.html) for the [schema import](../cockroachcloud/migrations-page.html).
+- [CSV](migrate-from-csv.html) for table data.
+- [Avro](migrate-from-avro.html) for table data.
+
+The storage media you use to export / import from can be intermediate data files or streaming data coming over the network. Options include:
+
+- Local "userdata" storage (see [`cockroach userdata`](cockroach-userfile-upload.html), [Use a Local File Server](use-a-local-file-server.html)).
+- Cloud blob storage (see [`IMPORT INTO`](import-into.html), [Import Performance Best Practices](import-performance-best-practices.html), [Use Cloud Storage](use-cloud-storage.html)).
+- Direct wire transfers (see [github.com/cockroachlabs/cdc-sink](https://github.com/cockroachlabs/cdc-sink) or [managed migration services](#managed-migration)).
+
+Database features for export/import from the source and target databases can include:
+
+- Tools for exporting from the source database may include: `pg_dump --schema-only` and `COPY`, `mysqldump`, `expdp`, etc.
+- For import into CockroachDB, use [`COPY`](copy-from.html) or [`IMPORT INTO`](import-into.html). For a bulk data migrations, most users should use `IMPORT INTO` because the tables will be offline anyway, and `IMPORT INTO` can [perform the data import much faster](import-performance-best-practices.html) than `COPY`.
+
+Decide which of the options above will meet your requirements while resulting in a process that is [restartable](#restartable) and [scalable](#scalable).
+
+<a name="restartable"></a>
+
+### Design a restartable process
+
+To have a higher chance of success, design your data migration process so it can be stopped and restarted from an intermediate state at any time during the process. This will help minimize errors and avoid wasted effort.
+
+Keep the following requirements in mind as you design a restartable import/export process:
+
```

Comment:
Somewhere we should talk about the fact that you can export and import the majority of your data before taking your source db down to drain such that you only have to deal with the incremental changes from your last import

Reply:
nice, thanks

added 

> - If possible, export and import the majority of your data before taking the source database down. This can ensure that you only have to deal with the incremental changes from your last import to complete the migration process.